README.md

```diff
@@ -1,284 +1,55 @@
+Please make sure to also read the documentation on [**citing superDARN and pydarn**](https://pydarn.readthedocs.io/en/latest/user/citing/). 
 
-fig, (ax1, ax2) = plt.subplots(2, 1)
-pydarn.RTP.plot_range_time(fitacf_data, parameter='elv',
-                           beam_num=7, ax=ax1, channel=1,
-                           date_fmt='')
-ax1.set_ylabel("ch. 1")
+As a quick tutorial on using pydarn to read a compressed file: 
+``` python
+import bz2
+import pydarn 
 
-pydarn.RTP.plot_range_time(fitacf_data, parameter='elv',
-                           beam_num=7, ax=ax2,
-                           channel=2)
-ax2.set_xlabel("Date (UTC)")
-ax2.set_ylabel("ch. 2")
+# read in compressed file
+fitacf_file = '20180220.C0.rkn.stream.fitacf.bz2'
+with bz2.open(fitacf_file) as fp: 
+      fitacf_stream = fp.read()
 
-fig.suptitle("RTP subplots using Axes Object")
-plt.show()
+reader = pydarn.SDarnRead(fitacf_stream, True)
+records = reader.read_fitacf()
 ```
-![pyDARN rang-time plot using Axes object](./pydarn_axes_range_time_plot.png)
-
-#### Turn on debugging 
 
-```python
-import pydarn
-import logging
+For more information and tutorials on pyDARN please see the [tutorial section](https://pydarn.readthedocs.io/en/latest/)
 
-pydarn_logger = logging.getLogger('pydarn').setLevel(logging.DEBUG)
+## Getting involved
 
-dmap_file = "./20180410.C0.sas.fitacf"
-damp_reader = pydarn.DmapRead(dmap_file)
-dmap_data = dmap_reader.read_records() 
-
-print(dmap_data[0]['origin.time'].value) 
-```
-Run the code and two log files will be produced:
-  * pydarn.log - DEBUG level info 
-  * pydarn_error.log - ERROR level info
+pyDARN is always looking for testers and developers keen on learning python, github, and/or SuperDARN data visualizations! 
+Here are some ways to get started: 
 
-### Release History 
+  - **Testing Pull Request**: to determine which [pull requests](https://github.com/SuperDARN/pydarn/pulls) need to be tested right away, filter them by their milestones.
+  - **Getting involved in projects**: if you are looking to help in a specific area, look at pyDARN's [projects tab](https://github.com/SuperDARN/pydarn/projects). The project you are interested in will give you the information on what is needed to get done, in progress, and to be reviewed. 
+  - **Answer questions**: if you want to try your hand at answer some pyDARN questions or adding to the discussion, look at pyDARN's [issues](https://github.com/SuperDARN/pydarn/issues) and filter by labels.
+  - **Become a developer**: if you want to practice those coding skills and at to the library, look at pyDARN [issues](https://github.com/SuperDARN/pydarn/issues) and filter by milestone's to see what needs to get done right away. 
 
-  * 0.0.1 
-    * Add: dmap DmapRead DmapWrite implemented.
-    * Add: superdarn DarnRead DarnWrite implemented
-    * Add: utils for converting dmap2dict and dict2dmap
+Please contact the leading developer or group if you would like to become a member of the team!
```

Comment:
Not sure what this means exactly. The person reading won't know who the leading developer or group is unless you put them down here :P

Reply:
Fixed